lib/http.js

```diff
@@ -309,6 +296,7 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
   app.post(
     routes.query,
     cors(),
+    validate('bedrock-edv-storage.postQuery'),
```

Comment:
should this be `postDocumentQuery` or the like?  We want to be fairly specific in naming these validators so we don't have collisions and folks looking at the validators in the file can get a good idea what they are used for by looking at the names.

Reply:
Naming here seems a bit tricky.  `postQuery` might be the least confusing. ;-)  If we also had a query validator for the query endpoint, we'd have `postQueryQuery`.  `postDocumentQuery` could be confused for a query validator on the documents endpoint vs a body validator on the document query endpoint..  And we don't prefix other endpoints with `postDocument*`, so it would be strange.  Perhaps just leave this alone as it does follow our pattern.  Add comments as appropriate to explain this particular edge case.

Reply:
@davidlehn that's for your insights.  Seems to me the root issue here is no well established convention, certainly not a documented one that I'm aware of.  But part of the convention would be to distinguish between `body` validators and `query` validators.

How does one extract any meaning at all from `postQuery`?  Is it a validator for the body of a query API for an unspecified `foo`?  Is it a validator for the URL query string for some unspecified `foo`.?

Reply:
The only convention we've had (which started ages ago) is prefix with get/post, no suffix for body validators, and add Query suffix for query validators.  I don't think we've had to deal with a "query" endpoint before.  Also using a "Body" suffix would help the situation, but we haven't done that anywhere else yet.  The `bedrock-validation` docs do have an example with the `Query` part.

If we can come up with a good convention, let's document and use it.

Another thing to help at call sites would be to enforce named params: `validate({body: '....postQuery', query: '...postQueryQuery'})`.  I think I orginally allowed that default body behavior as the `query` option was new, unused, and potentially rare to use, and it didn't seem worth breaking old code.  We could revisit that choice or at least be explicit when it's otherwise confusing.  That doesn't help where the validator is defined though.  Need explicit naming to fix that.

Reply:
What about `getDocumentQuery`? That follows the same convention as the next 2 `getEdvsQuery` and `getAuthorizationsQuery`.

Reply:
You can put query params on a URL for any method.

This all got complex!  Probably should just try for something like what we've been using in the past: `{namespace}.{method}{scope}{Query|Body}`, and use the named params `validate({[query: ...], [body: ...]})` syntax.  `scope` would just depend on what is used in that module namespace.  Hopefully we can keep modules simple enough that scope names don't get confusing.

Reply:
@davidlehn From your name logic listed, changing it to `postDocumentQuery` makes the most sense. I have updated the PR.